smart-contract-verifier/smart-contract-verifier-server/src/run.rs

```diff
@@ -0,0 +1,150 @@
+use crate::{
+    metrics::Metrics,
+    services::{
+        HealthService, SolidityVerifierService, SourcifyVerifierService, VyperVerifierService,
+    },
+    settings::Settings,
+    tracing::init_logs,
+};
+use actix_web::{dev::Server, App, HttpServer};
+use actix_web_prom::PrometheusMetrics;
+use smart_contract_verifier_proto::blockscout::smart_contract_verifier::v1::{
+    health_actix::route_health, health_server::HealthServer,
+    solidity_verifier_actix::route_solidity_verifier,
+    solidity_verifier_server::SolidityVerifierServer,
+    sourcify_verifier_actix::route_sourcify_verifier,
+    sourcify_verifier_server::SourcifyVerifierServer, vyper_verifier_actix::route_vyper_verifier,
+    vyper_verifier_server::VyperVerifierServer,
+};
+use std::{net::SocketAddr, sync::Arc};
+
+pub async fn run(settings: Settings) -> Result<(), anyhow::Error> {
+    init_logs(settings.jaeger);
+
+    let solidity_verifier = settings
+        .solidity
+        .enabled
+        .then(|| Arc::new(SolidityVerifierService::default()));
```

Comment:
settings aren't used here?

Reply:
Actual implementation of services will be in next PRs. So, for now we just use an empty struct as a template